components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.email/src/main/java/org/wso2/carbon/event/output/adapter/email/EmailEventAdapter.java

```diff
@@ -188,6 +198,19 @@ public void connect() throws ConnectionUnavailableException {
                         ("The adapter " + eventAdapterConfiguration.getName() + " " + msg);
             }
 
+            String replyTo = props.getProperty(EmailEventAdapterConstants.MAIL_SMTP_REPLY_TO);
+            if (replyTo != null) {
+
+                try {
+                    smtpReplyToAddress = InternetAddress.parse(replyTo);
+                } catch (AddressException e) {
+                    log.error("Error in retrieving smtp replyTo address : " + smtpFrom, e);
+                    String msg =
+                            "failed to connect to the mail server due to error in retrieving " + "smtp replyTo address";
+                    throw new ConnectionUnavailableException(
```

Comment:
Why do we need to log and throw ?

Reply:
In the upper layer [1] It handles only the ConnectionUnavailableException. And will not log the specific error related to this


[1].https://github.com/Buddhimah/carbon-analytics-common-1/blob/46331b3c35c2a691a1ea0231d588ebe1934fdda1/components/event-publisher/org.wso2.carbon.event.output.adapter.core/src/main/java/org/wso2/carbon/event/output/adapter/core/internal/OutputAdapterRuntime.java#L68

Reply:
Fix https://github.com/wso2/carbon-analytics-common/pull/734